src/applications/travel-pay/components/ClaimDetailsContent.jsx

```diff
@@ -110,21 +111,61 @@ export default function ClaimDetailsContent({
         </>
       )}
       <h2 className="vads-u-font-size--h3">Claim information</h2>
-      <p className="vads-u-font-weight--bold vads-u-margin-bottom--0">Where</p>
-      <p className="vads-u-margin-y--0">
-        {appointmentDate} at {appointmentTime} appointment
-      </p>
-      <p className="vads-u-margin-top--0">{facilityName}</p>
-      {claimsMgmtToggle &&
-        reimbursementAmount > 0 && (
-          <p>Reimbursement amount of ${reimbursementAmount}</p>
-        )}
+      {claimsMgmtToggle && (
+        <>
+          {(totalCostRequested > 0 || reimbursementAmount > 0) && (
```

Comment:
I think we need to check that these exist at all, unless you know otherwise. I'm thinking a claim in progress will have a totalCostRequested field, but won't (maybe) have a reimbursementAmount.

Reply:
While it's pretty javascript-y to be mixing types like this, I do think we'd still get the result we want without an explicit existence check:
```sh
> const a = null
undefined
> const b = undefined
undefined
> a > 0
false
> b > 0
false
```